lib/matplotlib/colorbar.py

```diff
@@ -1136,9 +1163,20 @@ def update_normal(self, mappable):
         Update solid patches, lines, etc.
 
         Unlike `.update_bruteforce`, this does not clear the axes.  This is
-        meant to be called when the image or contour plot to which this
-        colorbar belongs changes.
+        meant to be called when the norm of the image or contour plot to which
+        this colorbar belongs changes.
+
+        This resets the locator and formatter for the axis, so if these
+        have been customized, they will need to be customized again.
         """
+
+        _log.debug('update normal')
```

Comment:
```suggestion
        _log.debug('colorbar update normal')
```
Giving slightly more context helps understanding the log output.